src/main/java/com/ldtteam/structures/client/BlueprintChunk.java

```diff
@@ -64,7 +64,7 @@
      */
     public BlueprintChunk(final World worldIn, final int x, final int z)
     {
-        super(worldIn, new ChunkPos(x, z), new BiomeContainer(worldIn.registryAccess().registryOrThrow(Registry.BIOME_REGISTRY), new Biome[0]));
+        super(worldIn, new ChunkPos(x, z), new BiomeContainer(null, new Biome[0]));
```

Comment:
did you test this, won't this cause potential crashes?

Reply:
@Nightenom

Reply:
Btw, this should be solved with my changes. as I solved registryAccess() returning null.

Reply:
like this is 1.17, so probably no need to care xd, iirc in 1.16 this is not a thing

Reply:
I'll close the PR and port my fix over later today